com.unity.render-pipelines.high-definition/Runtime/Lighting/LightEvaluation.hlsl

```diff
@@ -295,6 +295,12 @@ SHADOW_TYPE EvaluateShadow_Directional( LightLoopContext lightLoopContext, Posit
         g_DebugShadowAttenuation = shadow;
 #endif
 
+    // Cloud shadows
```

Comment:
I'm not sure I understand why we need specific code for this.
Directional lights already support cookies. Seems redundant to me. Also what happens if a directional light has a cookie?

Reply:
so that's why it's a separate texture than the cookie, so that it's possible to use it together with a cookie

Reply:
Was this requested by artist? I assume that the vast majority of cases for directional cookie is to actually fake clouds. Would that make sense to make them mutually exclusive and save the extra sample?

Reply:
I guess you are right, this wasn't a request so let's do it directly with the cookie

Reply:
Maybe we can involve the artists with this one?
They might have another opinion :)
(even though I suppose all artists always want more :) )
Also we need to make sure that if we choose this path, we make it clear that enabling cloud layers overrides the regular directional light cookies.

Reply:
@sebastienlagarde ^^ ?